src/index/etc/mview.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<!--
+  ~ Copyright (c) Aligent Consulting. All rights reserved.
+  -->
+
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Mview/etc/mview.xsd">
+    <view id="fredhopper" class="Aligent\Fredhopper\Model\Indexer\ProductIndexer" group="indexer">
+        <subscriptions>
```

Comment:
Add `catalog_product_website` for website assignments/unassignments?

Do we also need `catalog_category_product` for category assignments/unassignments or would that be irrelevant or covered elsewhere?

Reply:
Hmmm - not sure how `catalog_product_website`  and `catalog_category_product` got missed - will add.